subjects/secondhalf/README.md

```diff
@@ -0,0 +1,41 @@
+## secondhalf
+
+### Instructions
+
+Write a function `SecondHalf()` that receives a slice of `int` and returns another slice of `int` with the last half of the values.
```

Comment:
suggestion: (...) with the second half of the values.

Reply:
It makes total sense, thank you. It is fixed.